doc/Customization-guide/How-to-contribute/Documentation-contribution-and-style-guide/index.md

```diff
@@ -11,62 +11,168 @@ sort_idx: 30
+
+## Tables
+* Introduce a table with a sentence that ends with a period, not a fragment that ends with a colon.
+* If a table is titled, an introductory sentence does not have to immediately precede the table.
+* Put the table title above the table.
+* Table dimensions must be visible on a minimum screen resolution, typically 800 pixels by 600 pixels.
+* Put footnote explanations at the end of the table, not at the bottom of the page.
+* Tables can be used to simulate frames.
+In this case, tables are better because older browsers cannot always process frames correctly.
+
+## Lists
+* Introduce a list with a heading or with a sentence or fragment ending with a colon.
+* Begin each entry in a bulleted or numbered list with a capital letter.
+* End each entry with a period if all entries are complete sentences, if they are a mixture of fragments and complete sentences, or if they all complete the introductory sentence or fragment.
+* Do not use ending punctuation for entries that are short phrases (three words or fewer) or single words.
+* If all entries are fragments that together with the introductory phrase do not form a complete sentence, do not end them with periods.
+* If you introduce a list with a fragment, do not treat the list and its introduction as one continuous sentence.
+That is, do not use semicolons or commas to end list items, and do not insert _and_ before the last list element.
+
+## Notes, tips, and cautions
+* Use notes sparingly so that they remain effective in drawing the user’s attention.
+* In general, try to use only one note in a Help topic.
+For example, if you must have two notes of the same type, such as a tip and a caution, combine them into one note with two paragraphs, or integrate one or both of the notes in the main text.
+* Never include two or more paragraphs formatted as notes without intervening text.
+If you need to put two notes together, format them as an unordered list within the note heading.
+* You can include lists within notes.
+* Use _note_ with the heading “Note” to indicate neutral or positive information that emphasizes or supplements important points of the main text.
+A note supplies information that may apply only in special cases.
+* Use _tip_ with the heading “Tip” to help users apply the techniques and procedures described in the text to their specific needs.
+A tip suggests alternative methods that may not be obvious and helps users understand the benefits and capabilities of the product.
+A tip is not essential to the basic understanding of the text.
+* Use _important note_ with the heading “Important” to provide information that is essential to the completion of a task.
+Users can disregard information in a note and still complete a task, but they should not disregard an important note.
+* Use _caution_ with the heading “Caution” to advise users that failure to take or avoid a specific action could result in loss of data, damage of hardware, and other issues of similar importance.
+
```

Comment:
Yes, I fixed this, but partially. To finalize it, I need to get the corresponding CSS classes from the Design team.